install/page_dbsettings.php

```diff
@@ -28,7 +28,7 @@
 
 $vars = & $_SESSION ['settings'];
 
-$func_connect = empty ( $vars ['DB_PCONNECT'] ) ? "mysql_connect" : "mysql_pconnect";
+$func_connect = empty ( $vars ['DB_PCONNECT'] ) ? "mysqli_connect" : "mysqli_pconnect";
```

Comment:
This needs to be changed as in the other file, once everything is working (I'm assuming the connection page is before the settings page).